geomstats/general_linear_group.py

```diff
@@ -7,29 +10,24 @@
 import geomstats.vectorization as vectorization
 
 
-class MatrixLieGroup(LieGroup):
+class GeneralLinearGroup(LieGroup):
     """
-    Base class for matrix Lie groups,
-    where each element is represented by a matrix by default.
+    Base class for the General Lie Group,
```

Comment:
sounds like a discrepency between name and comment? general linear group or lie group?